### DISCUSSION TITLE
Abstract to poster

### DISCUSSION THREAD

**Discussion by RobeReyes** [will-henney/globule-seminario#33] [General]
This is a first idea to the abstract:

De las imágenes del JWST y del HST en $H\alpha$ se pueden apreciar con mucho detalle la presencia de cúmulos globulares en la nebulosa M1-67 de la estrella WR-124.  A partir de estas observaciones queremos saber como es la interacción entre la presión RAM de la estrella y la presión interna debido al flujo foto-evaporativo de estos cúmulos, durante el estudio de estas interacciones se obtuvieron algunos parámetros físicos en las cáscaras chocadas considerando un  modelo estacionario y se pudo conocer mejor la distribución espacial tomando en cuenta el ángulo que tienen los cumulos entre el plano del cielo y el eje de simetría de estos cúmulos.

**Comment by will-henney** [comment]
I think this is a good start! I will put it in English and maybe make a few additions. 

One comment would be that "cúmulos globulares" are "globular clusters", which are something very different from the gaseous molecular globules that we are studying. Clusters in astronomy always refers to groups of stars

**Comment by will-henney** [comment]
Here is my first pass at translating Roberto's draft, with some small changes. 

>  From the JWST and HST images in H$\alpha$, the presence of dense globules in the M1-67 nebula of the star WR-124 can be seen in great detail. We wish to use these  observations in order to study the interaction between the RAM pressure of the stellar wind from the WR star and the internal pressure due to the photoevaporative flow of these globules.  By considering a steady state hydrodynamic model for these interactions, we obtain physical parameters of the collision shells and de-project the spatial distribution of globules, finding that the majority are concentrated in a narrow range of radii: 0.5 +/- 0.1 pc from the star.

I will look at it again tomorrow

**Comment by will-henney** [comment]
We also need a title for the poster. @RobeReyes would you like to propose one?

**Comment by RobeReyes** [comment]
i don't  know what could be a good title, i only think like:
- interaction for globules with WR 124 in M1-67
- distribution of globules qround WR 124

Maybe you can find another title.

**Comment by will-henney** [comment]
OK, then I suggest the following:

> Shielded from the wind: photoevaporating neutral globules around WR 124

What do you think?   The idea is that the stellar wind does not interact directly with the neutral globule, but only indirectly via the ionized ablation flow.

**Comment by RobeReyes** [comment]
okay, i send the title:

> Shielded from the wind: photoevaporating neutral globules around WR 124

with tje abstract:

> From the JWST and HST images in H$\alpha$, the presence of dense globules in the M1-67 nebula of the star WR-124 can be seen in great detail. We wish to use these observations in order to study the interaction between the RAM pressure of the stellar wind from the WR star and the internal pressure due to the photoevaporative flow of these globules. By considering a steady state hydrodynamic model for these interactions, we obtain physical parameters of the collision shells and de-project the spatial distribution of globules, finding that the majority are concentrated in a narrow range of radii: 0.5 +/- 0.1 pc from the star.

**Comment by will-henney** [comment]
Actually, I have a few more changes to make to the abstract to make it more accessible. Could you wait a few minutes and I will post a new version.

**Comment by RobeReyes** [comment]
okay i will wait

**Comment by will-henney** [comment]
OK, how about this?  I added more introductory text to describe the context of the work. 

> The circumstellar nebula M1-67 around the Wolf-Rayet star WR~124
> contains hundreds of small neutral globules, as revealed by recent
> JWST images.  However, despite the powerful stellar wind from the
> star, the globules do not interact directly with the wind.  Instead,
> they are hydrodynamically shielded by a transonic warm ionized flow
> away from their surfaces that is induced by the Lyman continuum
> radiation of the star.  This inwardly directed photoevaporation flow
> shocks against the outflowing stellar wind to form dense
> hemispherical ionized shells that are a few times larger than the
> globules and which contribute a significant fraction of the
> recombination line luminosity of the nebula.  We analyze archival
> HST H$\alpha$ images of the nebula 
> in order to study the balance between the ram pressure of the stellar wind and the 
> internal pressure due to the photoevaporative flows.  By considering a steady state
> hydrodynamic model for these interactions, we obtain physical parameters of the 
> collision shells and de-project the spatial distribution of globules, 
> finding that the majority are concentrated in a 
> narrow range of radii: 0.5 +/- 0.1 pc from the star.

**Comment by RobeReyes** [comment]
to send to Jesus i think that the first short abstract its ok. I ask to him in the morning and he told me that with a brief intoduction is fine.

The last introduction could be for  the poster.

What do you think?
